Dockerfile

```diff
@@ -45,7 +49,7 @@ VOLUME /config /transcode
 ENV CHANGE_CONFIG_DIR_OWNERSHIP="true" \
     HOME="/config"
 
-ARG TAG=beta
+ARG TAG=public
```

Comment:
Any reason for this change?

Reply:
Indeed, there isn't. Actually the Download API doesn't really care about this tag, it returns the same file for both Beta and Public. If I suggested to use Public was in case a new Beta package was released and suddenly makes Plex not working on Docker. But that should not happen, so we can stay with Beta.